extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -558,8 +558,7 @@ export class AmpA4A extends AMP.BaseElement {
     // its element ancestry.
     if (!this.isValidElement()) {
       // TODO(kjwright): collapse?
-      user().warn(TAG, this.element.getAttribute('type'),
-          'Amp ad element ignored as invalid', this.element);
+      user().warn(TAG, '%s','Amp ad element ignored as invalid',this.element, this.element.getAttribute('type'));
```

Comment:
here's an example of a pretty stupid transform. the actual message probably needs to be fixed.